.github/workflows/main.yml

```diff
@@ -91,10 +91,11 @@ permissions:
 jobs:
   build-graalvm-linux:
     name: /${{ matrix.env.PRIMARY }} ${{ matrix.env.GATE_TAGS }} JDK${{ matrix.env.JDK_VERSION }}
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
```

Comment:
Thanks for suggesting this but considering that build do consume quite some resources (11 jobs, 3h in total, almost 6000 workflow runs), I think it would be better if we keep testing on just one version of Ubuntu here.

Reply:
Ah yes I see, thank you for the feedback, let me take it out :smile: